test/interpreter/kernel/exec/numeric-instructions.js

```diff
@@ -251,6 +270,56 @@ describe("kernel exec - numeric instructions", () => {
       resEqual: new f32(2)
     },
 
+    {
```

Comment:
@xtuc First test cases are just below here. They cover `nan + value`, `nan + nan` and `value + nan`. I will add more.